dev-util/bloaty/bloaty-1.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="A size profiler for binaries"
+HOMEPAGE="https://github.com/google/bloaty"
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/google/${PN}"
+	IUSE+=" test"
+	RESTRICT="!test? ( test )"
+else
+	SRC_URI="https://github.com/google/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+	KEYWORDS="~amd64"
+fi
+
+BDEPEND="
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-libs/capstone:=
+	dev-libs/protobuf:=
+	dev-libs/re2:=
+"
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DBLOATY_ENABLE_CMAKETARGETS=OFF
+		-DBUILD_SHARED_LIBS=OFF
+		-DCMAKE_SKIP_RPATH=ON
+	)
+	if [[ ${PV} == 9999 ]]; then
+		mycmakeargs+=(
+			-DBUILD_TESTING=$(usex test)
+			$(usex test -DINSTALL_GTEST=OFF "")
+		)
+	fi
```

Comment:
Are you sure these are not enabled on 1.1 release? Didn't download tarball but by browsing files on release tag, they seem to be present.

Reply:
Tests are not part of the release tarball, they can only be run when cloning from git, that's why it's only enabled in 9999. Regarding the gtest instalation issue, I sent a PR to fix it upstream, but it was only merged after the 1.1 release, that's why the INSTALL_GTEST=OFF is necessary here.